leap-consent-ui/src/main/resources/vaadinapp_es.properties

```diff
@@ -864,3 +864,29 @@ UserPreferences-emergency_contact=Contacto de Emergencia
 UserPreferences-emergency_contact_phone=Teléfono del Contacto de Emergencia
 UserPreferences-relationship=Relación
 UserPreferences-image=Imagen del Perfil
+# PatientPrivacyQuestionnaire #
+PatientPrivacy-questionnaire_response_item_1_1=Usar opción predeterminada
+PatientPrivacy-questionnaire_response_item_1_2=Opción de fecha personalizada
+PatientPrivacy-questionnaire_response_item_1_3_1=24 horas
+PatientPrivacy-questionnaire_response_item_1_3_2=1 año
+PatientPrivacy-questionnaire_response_item_1_3_3=5 años
+PatientPrivacy-questionnaire_response_item_1_3_4=10 años
+PatientPrivacy-questionnaire_response_item_1_3_5=Fecha y hora de inicio calculadas según la selección de opciones
+PatientPrivacy-questionnaire_response_item_1_3_6=Fecha y hora de finalización calculadas según la selección de opciones
+PatientPrivacy-questionnaire_response_item_1_4_1=Fecha y hora de inicio
+PatientPrivacy-questionnaire_response_item_1_4_2=Fecha y hora de finalización
+PatientPrivacy-questionnaire_response_item_2_1=Denegar el acceso a lo siguiente
+PatientPrivacy-questionnaire_response_item_2_2=Permitir que se intercambien todo tipo de datos
+PatientPrivacy-questionnaire_response_item_2_1_1=Seleccionar de la lista
+PatientPrivacy-questionnaire_response_item_3_1=Facultativo
+PatientPrivacy-questionnaire_response_item_3_1_1=Practicante - Custodio (Seleccione de la lista).
+PatientPrivacy-questionnaire_response_item_3_2=Organización
+PatientPrivacy-questionnaire_response_item_3_2_1=Organización - Custodio (Seleccionar de la lista).
+PatientPrivacy-questionnaire_response_item_4_1=Facultativo
+PatientPrivacy-questionnaire_response_item_4_1_1=Practicante - Destinatario (Seleccionar de la lista).
+PatientPrivacy-questionnaire_response_item_4_2=Organización
+PatientPrivacy-questionnaire_response_item_4_2_1=Organización - Destinatario (Seleccionar de la lista).
+PatientPrivacy-questionnaire_response_item_5_1=Quitarlos
+PatientPrivacy-questionnaire_response_item_5_2=No tengo preocupaciones de privacidad
+PatientPrivacy-questionnaire_response_item_5_1_1=Seleccionar todo: esto eliminará toda la información marcada como confidencial.
+PatientPrivacy-questionnaire_response_item_6=Este último paso capturará su firma y creará un pdf legible por humanos de este consentimiento. (Firma electrónica capturada)
```

Comment:
Please change this to: 
Este último paso capturará su firma y creará un pdf legible de este consentimiento. (Firma electrónica capturada)

Reply:
thanks done